devito/function.py

```diff
@@ -256,9 +256,12 @@ def staggered(self):
     @cached_property
     def shape(self):
         """
-        Shape of the domain associated with this :class:`TensorFunction`.
-        The domain constitutes the area of the data written to in a
-        stencil update.
+        Shape of the domain region. The domain constitutes the area of the
+        data written to by an :class:`Operator`.
```

Comment:
Is it really the write only region or the read/write one?

Reply:
write; normally, when you got a stencil, you read from the halo region as well